posthog/migrations/0255_user_prompt_sequence_state.py

```diff
@@ -0,0 +1,33 @@
+# Generated by Django 3.2.14 on 2022-08-04 19:38
+
+import django.db.models.deletion
+import django.utils.timezone
+from django.conf import settings
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ("posthog", "0254_prompt_sequence_state"),
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name="UserPromptSequenceState",
+            fields=[
+                ("id", models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name="ID")),
+                ("key", models.CharField(max_length=400)),
+                ("last_updated_at", models.DateTimeField(default=django.utils.timezone.now)),
+                ("step", models.IntegerField(default=0)),
+                ("completed", models.BooleanField(default=False)),
+                ("dismissed", models.BooleanField(default=False)),
+                ("user", models.ForeignKey(on_delete=django.db.models.deletion.CASCADE, to=settings.AUTH_USER_MODEL)),
+            ],
+        ),
+        migrations.DeleteModel(name="PromptSequenceState",),
```

Comment:
Code looks alright to me, although this specific migration can give you problems. Before this goes in, ensure that:

1. The code before-this-PR can run when this migration has run (to prevent downtime on cloud) (Imagine a situation on cloud where the migration has run, but the code hasn't been updated yet. This happens because multiple web instances.)
2. Running this migration doesn't lock other tables which brings down other parts of the app querying those tables.

Reply:
Thanks, I'll consult with other people to be sure the migration doesn't do any of that 🙏

Reply:
s/works/doesn't crash PostHog - it's _fine_ if it returns nothing for the 10-15 minutes it takes to completely get on the new version, since this is new / not mission critical / doesn't have much data yet.